eo-runtime/src/test/eo/org/eolang/cage-tests.eo

```diff
@@ -29,120 +29,89 @@
 
 [] > writes-into-cage
   [z] > a
-  cage 0 > x
-  nop > @
-    assert-that
-      seq
-        x.write
-          a 7
-        x.z
-      $.equal-to 7
+  cage a > x
+  assert-that > @
+    seq
+      x.write
+        a 7
+      x.z
+    $.equal-to 7
 
 [] > avoid-infinite-loop
-  cage 0 > x
+  cage > x
+    TRUE.if
   x' > copy
-  nop > @
-    assert-that
-      seq
-        x.write 42
-        copy.<
-        x.write (7.plus copy)
-        x
-      $.equal-to 49
-
-[] > avoid-infinite-loop-second-case
```

Comment:
@maxonfjvipon Why did you remove this tests case?

Reply:
@Graur I don't see the way it can be refactored since we can't write `pair` object to the `cage` at the beginning and then write `add` object

Reply:
@Graur in other words this and other removed tests are no longer relevant